spotbugs-exclude.xml

```diff
@@ -26,6 +26,7 @@
     <Or>
       <Bug pattern="RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT"/>
       <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
+      <Bug pattern="DE_MIGHT_IGNORE"/>
```

Comment:
What warning are you suppressing here?

Reply:
It doesn't need to be suppressed anymore, removing.